project/types/types.go

```diff
@@ -0,0 +1,29 @@
+package types
```

Comment:
What is the package was options.  The everything would be `options.Up`, `options.Build`.  Seems prettier to me.